app/Card.js

```diff
@@ -44,6 +44,7 @@ module.exports = class Card {
     this.card.appendChild(this.header);
     context.appendChild(this.card);
     if (modal) this.toggleDraggable();
+    this.toggleDroppable();
```

Comment:
The addition of `this.toggleDroppable();` on a separate line from `if (modal) this.toggleDraggable();` creates a situation in which every `Card` instance, whether it has the `modal` constructor option enabled or not, will be draggable. This defeats the purpose of even having a `modal` constructor option.